package-lock.json

```diff
@@ -16,8 +16,15 @@
         "@babel/core": "^7.16.5",
         "@babel/preset-env": "^7.16.5",
         "@babel/preset-react": "^7.16.5",
+        "autoprefixer": "^10.4.1",
```

Comment:
Nice job! Just need to add new scripts to transpile css to package.json file and modify the existing ones who need it.